inst/ms/aggreCAT.qmd

```diff
@@ -42,11 +63,22 @@ bibliography: bibliography.bib
 editor: 
```

Comment:
Abstract comments:
- SCORE program is only 3000 claims for phase one TA2. Over the whole project there are 4000 single trace and 100 bushel papers that get scores from TA2, and ~30,000 claims rated by TA3 teams
- Has the count of 28 aggregators been updated for the current package design?

Reply:
- [x] Updated reference to claim number
- [ ] Count not yet updated!! @egouldo to update

Reply:
- AverageWAgg
  - ArMean
  - Median
  - GeoMean
  - LOArmean
  - ProvitArMean
- BayesianWAgg
  - BayTriVar
  - BayPriorsAgg
- DistributionWagg
  - DistribArMean
  - TriDistribArMean
- ExtremisationWAgg
  - BetaArMean
  - BetaArMean2
- IntervalWAgg
  - IntWAgg
  - IndIntWAgg
  - AsymWAgg
  - IndIntAsymWAgg
  - VarIndIntWAgg
  - KitchSinkWAgg
- LinearWAgg
  - DistLimitWAgg
  - GranWAgg
  - OutWAgg
- REasoningWAgg
  - ReasonWAgg
  - ReasonWAgg2 
- ShiftingWAgg
  - ShiftWAgg
  - BestShiftWAgg
  - IntShiftWAgg
  - DistShiftWAgg
  - DistIntShiftWAgg

That's 27 specific methods we have coded, if I've counted correctly.
AND, if user-supplied weights are provided to LinearWAgg, then I guess we have 28 methods. What do you think @Doi90 , what number should I go with?

Reply:
Technically the two ReasonWAggs have BetaReasonWAgg versions too with the extremisation argument, and there are two different user supplied weight options. I think you're fine going with 27 + 2 user though.

Reply:
I've just gone with 27 in the paper, and haven't expanded on the '+2'.